musica/musica.meta

```diff
@@ -1,22 +1,11 @@
 [ccpp-table-properties]
-  name = micm
+  name = musica
   type = scheme
+  dynamic_constituent_routine = musica_register
 
 [ccpp-arg-table]
-  name  = micm_init
+  name  = musica_init
```

Comment:
I think you need a musica_register metadata section

Reply:
I believe for the dynamic species function, the typical meta data is replaced by this line:
https://github.com/mattldawson/atmospheric_physics/blob/6fb36e8b6356af8e50ab0df1a6b8afd0f4f29318/musica/musica.meta#L4

but I will defer to @peverwhee on this